frontend/packages/dev-console/src/components/pipelines/modals/triggers/TriggerTemplateSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { useField } from 'formik';
+import { DropdownField } from '@console/shared/src';
+import { Pipeline } from '../../../../utils/pipeline-augment';
+import { usePipelineTriggerTemplateNames } from '../../utils/triggers';
+
```

Comment:
nit: remove this extra line

Reply:
I feel it makes it easier to not break the import sort rule by putting the scss one line away from the JS imports. It has to go last by our rules, but it's not enforced. So it's easier to see when imports are incorrect imo.

Reply:
Hmm mentioned it since in our codebase we have kept scss files at the end but without any line breaks. Anyways your point makes sense.

Reply:
This is the only criteria I am aware of:

> Absolute imports should come first then relative
>     - SCSS should be the last import

If you think the spirit of this comment means that we shouldn't have a space, I'd be happy to adjust.